### PR TITLE
Bugfix nlu comparison persist/load nlg responses

### PR DESCRIPTION
**Pull request by melindaloubser1**
**Proposed changes**:
- fixes https://github.com/RasaHQ/rasa/issues/5201
- persist NLG responses along with NLU training data during NLU pipeline comparison 

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by melindaloubser1** [comment]
@dakshvar22 `test_nlu_comparison` passes; should I edit that test to include NLG responses or add a separate test?

File: tests/nlu/base/test_evaluation.py [hunk 1]
```diff
@@ -716,12 +716,16 @@ def test_get_evaluation_metrics(
...
         NLU_DEFAULT_CONFIG_PATH,
-        "sample_configs/config_supervised_embeddings.yml",
+        "sample_configs/config_convert_response_selector.yml",
     ]
     output = tmpdir.strpath
 
     compare_nlu_models(
-        configs, DEFAULT_DATA_PATH, output, runs=2, exclusion_percentages=[50, 80]
+        configs,
+        "data/test_evaluations/nlu_nlg",
```
Comment (dakshvar22) [inline tests/nlu/base/test_evaluation.py]: Can you assign this path to a constant variable. Easier to maintain.

Comment (melindaloubser1) [reply tests/nlu/base/test_evaluation.py]: sure

**Comment by melindaloubser1** [comment]
replaced by https://github.com/RasaHQ/rasa/pull/5294

**Pull request closed without merging**